app/controllers/helper/ShapefilesCreatorHelper.java

```diff
@@ -31,6 +31,16 @@
  */
 public class ShapefilesCreatorHelper {
 
+ /* This class handles the creation of Shapefile archives to be used by the SidewalkAPIController.
```

Comment:
Not sure what happened here, but doesn't look intentional. Make sure to do a `git diff` before you commit code so that you know that you aren't accidentally committing things you didn't intend to!

Reply:
I am not able to edit this in my local!

Reply:
I just removed it